src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. In case the main font doesn't load, the browser will use the fallback font. For example: `font-family: Roboto, Arial, sans-serif;`.